packages/metrics/src/config/EnvironmentVariablesService.ts

```diff
@@ -13,12 +13,27 @@ class EnvironmentVariablesService
 {
   private namespaceVariable = 'POWERTOOLS_METRICS_NAMESPACE';
 
+  private readonly disabledVariable = 'POWERTOOLS_METRICS_DISABLED';
+
   /**
    * Get the value of the `POWERTOOLS_METRICS_NAMESPACE` environment variable.
    */
   public getNamespace(): string {
     return this.get(this.namespaceVariable);
   }
+
+  /**
+   * Get the value of the `POWERTOOLS_METRICS_DISABLED` environment variable.
+   */
+  public getMetricsDisabled(): boolean | undefined {
+    const value = this.get(this.disabledVariable);
+
+    if (this.isValueTrue(value)) return true;
+
+    if (this.isValueFalse(value)) return false;
+
+    return undefined;
+  }
```

Comment:
I need to make this return `boolean | undefined` to account for not setting this value.

Reply:
What if instead of this, we extract the logic that we currently have in `setDisabled`  from the Metrics class, so that we always return a boolean?

We have precedents (like [this](https://github.com/aws-powertools/powertools-lambda-typescript/blob/12f3e448d8104b08518370a71db12646dededa4f/packages/logger/src/config/EnvironmentVariablesService.ts#L100-L105)) of methods in the env variable config service that look at multiple env variables.

What do you think?

Reply:
Like this? 

```suggestion
  public getMetricsDisabled(): boolean {
    const devMode = this.isDevMode();
    const value = this.get(this.disabledVariable);

    if (this.isValueFalse(value)) return false;
    if (this.isValueTrue(value)) return true;
    if(this.isDevMode()) return false;

    return false;
  }
```

It would consolidate the env var logic at one place, which is nice.